lib/oli_web/live/delivery/remix/hide_resource_modal.ex

```diff
@@ -0,0 +1,55 @@
+defmodule OliWeb.Delivery.Remix.HideResourceModal do
+  use Phoenix.LiveComponent
+  use Phoenix.HTML
+
+  alias Oli.Delivery.Hierarchy.HierarchyNode
+
+  def render(%{node: %HierarchyNode{}} = assigns) do
+    ~H"""
+    <div
+      class="modal fade show"
+      id={"hide_#{@node.uuid}"}
+      tabindex="-1"
+      role="dialog"
+      aria-hidden="true"
+      phx-hook="ModalLaunch"
+    >
+      <div class="modal-dialog" role="document">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h5 class="modal-title">
+              <%= get_label_action(@node) |> String.capitalize() %>
+              <%= @node.revision.title |> String.capitalize() %>
+            </h5>
+            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close">
+            </button>
+          </div>
+          <div class="modal-body">
+            Are you sure you want to <%= get_label_action(@node) %> <b><%= @node.revision.title %></b>?
+          </div>
+          <div class="modal-footer">
+            <button
+              type="button"
+              class="btn btn-secondary"
+              data-bs-dismiss="modal"
+              phx-click="HideResourceModal.cancel"
+            >
+              Cancel
+            </button>
+            <button
+              phx-click="HideResourceModal.toggle"
+              phx-key="enter"
+              phx-value-uuid={@node.uuid}
+              class="btn btn-danger"
```

Comment:
Going forward, we should avoid introducing new buttons using any bootstrap classes, e.g. `btn btn-danger`. Instead, we should be using tailwind and for higher-level common components such as buttons, make use of the `<.button />` component defined in `OliWeb.Components.Common`

Reply:
Thanks for you review @eliknebel. I'll change those buttons to use the button component.